### DISCUSSION TITLE
Vote for a logo of iceberg-rust.

### DISCUSSION THREAD

**Discussion by liurenjie1024** [apache/iceberg-rust#523] [Polls]
Following some previous discussion about iceberg-rust's new logo(1, 2), we want to start a vote for the logo. The candidates are from [2](https://github.com/apache/iceberg-rust/discussions/516) and the authors have agreed to donate them to the ASF if choosen.

# Candidates

Option 1:  
![310691257-ea2a8b46-15fb-4aba-9a06-49c58bc0b821](https://github.com/user-attachments/assets/6b15c2b2-9f24-49f3-a373-e2959ba0c28b)

Option 2:

![316239345-0e315b79-373f-412c-9117-e4cb6b00f7b1](https://github.com/user-attachments/assets/39728f06-d112-4391-8f96-85ca2bc7febf)

# Reference

1. https://apache-iceberg.slack.com/archives/C05HTENMJG4/p1708425331928999
2. https://github.com/apache/iceberg-rust/discussions/516

**Comment by Xuanwo** [comment]
Hi, I love Option 2. However I think we need some polish if we want to use this:

- The snowflake on the crab is too similar to [Snowflake](https://www.snowflake.com/en/), a vendor within our iceberg community.
- We need a space to include `Apache`.
- We should add a suffix like `TM` after `iceberg`.

*I'm willing to allocate a designer to handle that work if needed.*

**Reply by liurenjie1024** [reply]
> The snowflake on the crab is too similar to [Snowflake](https://www.snowflake.com/en/), a vendor within our iceberg community.

I don't have such strong feelings, I think it's more close to ice rather than snow.

> We need a space to include Apache.

+1.

> We should add a suffix like TM after iceberg.

+1. 

> I'm willing to allocate a designer to handle that work if needed.

That would be awesome!

**Reply by liurenjie1024** [reply]
> Hi, I love Option 2. 

Don't forget add your vote.

**Reply by Xuanwo** [reply]
It's highly likely that Option 2 will be chosen. I have spoken with the founder of Databend, @BohuTANG, and he is willing to have a designer create a revised design based on this logo.

I will keep update in this thread.

**Comment by Fokko** [comment]
Thanks Xuanwo for offering help.

My preference is also towards 2, I would love to see a bit more cartoony
style. Like the Docker logo, I think Ferris can play a good role there. As
already said earlier in the thread, there should not be any similarities in
the logo with any vendor.

One side note, an official vote to adopt this should be run on the devlist.

Kind regards,
Fokko





Op wo 7 aug 2024 om 09:07 schreef Xuanwo ***@***.***>:

**Reply by Xuanwo** [reply]
Great! I think we can finalize the vote here (while 72 hours reached) and start a new one for our revised logo once our designer has completed her work.

**Reply by liurenjie1024** [reply]
> Great! I think we can finalize the vote here (while 72 hours reached) and start a new one for our revised logo once our designer has completed her work.

+1.
